pages/content/pixi/info-texts/mobile-friendliness.md

```diff
@@ -3,3 +3,5 @@ $title: Mobile-friendliness
 ---
 
 Having a mobile-friendly website is a critical part of your online presence. In many countries, smartphone traffic now exceeds desktop traffic. If you haven't made your website mobile-friendly, you should! [Learn how](https://support.google.com/webmasters/answer/6352293#blocked-resources).
+
+In the event of analysis failure, please manually check your [mobile-friendly status](https://search.google.com/test/mobile-friendly).
```

Comment:
is there any way to hide this information until a failure has occurred?

Reply:
Not easily at the moment because we would have to resolve dynamic state (response results) with static state (jinja template). @matthiasrohmer and @robinvanopstal were going to look at this tomorrow with #4556, is it OK if we merge as is until then?

Reply:
How's this instead?
```suggestion
To run the test directly, use the Google Search Console [Mobile-Friendly Test](https://search.google.com/test/mobile-friendly).
```